jdaviz/app.py

```diff
@@ -1300,67 +1300,68 @@ def _update_selected_data_items(self, viewer_id, selected_items):
         viewer_item = self._viewer_item_by_id(viewer_id)
         viewer = self._viewer_by_id(viewer_id)
 
-        if isinstance(selected_items, list):
-            selected_items = {si: 'visible' for si in selected_items}
-        # Update the stored selected data items
-        viewer_item['selected_data_items'] = selected_items
+        if viewer is not None:
```

Comment:
Isn't it less diff if you do this instead? Then you don't have to indent so much.

```python
if viewer is None:
    return
```